app/uk/gov/hmrc/tai/model/domain/IabdDetails.scala

```diff
@@ -31,9 +33,55 @@ case class IabdDetails(
   captureDate: Option[LocalDate]
 )
 
-object IabdDetails extends IabdTypeConstants {
+object IabdDetailsToggleOff extends IabdTypeConstants {
+  private val dateReads: Reads[LocalDate] = localDateReads("dd/MM/yyyy")
+
+  implicit val reads: Reads[IabdDetails] = { (json: JsValue) =>
+    val nino = (json \ "nino").asOpt[String]
+    val employmentSequenceNumber = (json \ "employmentSequenceNumber").asOpt[Int]
+    val source = (json \ "source").asOpt[Int]
+    val iabdType = (json \ "type").asOpt[Int]
+    val receiptDate = (json \ "receiptDate").asOpt[LocalDate](dateReads)
+    val captureDate = (json \ "captureDate").asOpt[LocalDate](dateReads)
+    JsSuccess(
+      IabdDetails(nino, employmentSequenceNumber, source, iabdType, receiptDate, captureDate)
+    )
+  }
```

Comment:
Weird syntax.

See:
`implicit val reads: Reads[HipIabdUpdateRequestBody] = (
    (JsPath \ "currentOptimisticLock").read[Int] and
      (JsPath \ "grossAmount").read[BigDecimal] and
      (JsPath \ "netAmount").readNullable[BigDecimal]
  )(HipIabdUpdateRequestBody.apply _)`